.rustfmt.toml

```diff
@@ -1 +1,2 @@
 hard_tabs=true
+merge_imports=true
```

Comment:
Ah so rustfmt does have something to merge imports? cool!

(I didn't add that line. Well.. I can't remember I did it)

Reply:
I added this in the rococo branch.

Why does this contains rococo changes?

Reply:
Because we also need to get rococo changes on cumulus branch right?